File: seccion_8/ej.py
class Prueba:
    def __init__(self, nombre):
        self.nombre = nombre
        print("Objeto creado:", self.nombre)

p1 = Prueba("Objeto1")
p2 = Prueba("Objeto2")
p3 = Prueba("Objeto3")
p2 = None
p3 = None
print("Fin del programa")

File: seccion_8/ej.java
class Prueba {
 String nombre;
 Prueba(String nombre) {
 this.nombre = nombre;
 }
}
public class ej {
 public static void main(String[] args) {
 Prueba p1 = new Prueba("Objeto1");
 Prueba p2 = new Prueba("Objeto2");
 Prueba p3 = new Prueba("Objeto3");
 p2 = null;
 p3 = null;
 System.gc();
 System.out.println("Fin del programa");
 }
}
